generator/generator.yml

```diff
@@ -101,3 +101,41 @@ modules:
       - 1.3.6.1.4.1.1718.3.2.2  # infeedTable
       - 1.3.6.1.4.1.1718.3.2.3  # outletTable
 
+# A10 Ax Load Balancers
+#
+# The MIB files are available for download through the load balancer GUI:
```

Comment:
I'm not sure we want this to be a strict requirement.  There are many useful MIBs that require accounts, or sometimes paywalls.

What about having a separate `generator_a10.yml` and output yml file?